stock_picking_invoice_link/models/account_move.py

```diff
@@ -16,8 +16,11 @@ class AccountMove(models.Model):
         string="Related Pickings",
         store=True,
         compute="_compute_picking_ids",
-        help="Related pickings "
-        "(only when the invoice has been generated from a sale order).",
+        help="Related pickings (only when the invoice has been generated from a sale order).",
+    )
+
+    delivery_count = fields.Integer(
```

Comment:
`picking_count`, as this is also reuse for purchase pickings on module `purchase_stock_picking_invoice_link`.